packages/backend/src/services/UnfurlService/UnfurlService.ts

```diff
@@ -145,7 +296,7 @@ export class UnfurlService {
             });
             try {
                 await page.goto(url, {
-                    timeout: 150000, // Wait 2.5 mins for the page to load
+                    timeout: 300000, // Wait 2.5 mins for the page to load
```

Comment:
why did you increase this ?

Reply:
oh my bad this was just debugging, i'll revert it